backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/ImportSchemaTask.java

```diff
@@ -36,35 +36,31 @@ public void run() {
             this.add(metadataTask);
             metadataTask.run();
 
-            // in case of emx1
             boolean skipped = true;
-            if (store.containsTable("attributes")) {
 
-            } else {
-
-              // create task for the import, including subtasks for each sheet
-              for (Table table : s.getTablesSorted()) {
-                if (store.containsTable(table.getName())) {
-                  ImportTableTask importTableTask = new ImportTableTask(store, table, isStrict());
-                  this.add(importTableTask);
-                  importTableTask.run();
-                  skipped = false;
-                }
+            // create task for the import, including subtasks for each sheet
+            for (Table table : s.getTablesSorted()) {
+              if (store.containsTable(table.getName())) {
+                ImportTableTask importTableTask = new ImportTableTask(store, table, isStrict());
+                this.add(importTableTask);
+                importTableTask.run();
+                skipped = false;
               }
+            }
 
-              // warn for unknown sheet names
-              Collection<String> tableNames = s.getTableNames();
-              for (String sheet : store.tableNames()) {
-                if (!"molgenis".equals(sheet)
-                    && !"molgenis_settings".equals(sheet)
-                    && !"molgenis_members".equals(sheet)
-                    && !tableNames.contains(sheet)) {
-                  this.step(
-                          "Sheet with name '"
-                              + sheet
-                              + "' was skipped: no table with that name found")
-                      .skipped();
-                }
+            // warn for unknown sheet names
+            Collection<String> tableNames = s.getTableNames();
+            for (String sheet : store.tableNames()) {
+              if (!sheet.startsWith("_files/")
```

Comment:
Need to this to api docs

Reply:
Not really needed because table names with _ are not allowed.
But I will add a note on structure of csv.zip file